qa/rpc-tests/blockdelay_2.py

```diff
@@ -19,14 +19,9 @@
 import time
 class blockdelay_2(BitcoinTestFramework):
 
-    alert_filename = None
```

Comment:
This alert file code can be removed also from following Python modules: `block_delay.py` (not used at all, being in comment section) and `forknotify.py` (passed to nodes as startup parameter, but actually never used)